backend/migrations/versions/b405d05fd250_.py

```diff
@@ -1,16 +1,16 @@
 """empty message
 
-Revision ID: 73c34860b3f0
```

Comment:
why do you need to change this?

Reply:
I think that change is going to break the existing databases. If that is the case, you should mention that at least in the description of the PR to notify the colleagues.

Reply:
its another document. github thinks i changed old version, but i deleted all migratons and created new, because flask migrate don't work with unique constraint upgrade or downgrade.